test/test-udp-send-unreachable.c

```diff
@@ -138,12 +155,45 @@ TEST_IMPL(udp_send_unreachable) {
                   send_cb);
   ASSERT(r == 0);
 
+  r = uv_udp_init(uv_default_loop(), &client2);
+  ASSERT(r == 0);
+
+  r = uv_udp_bind(&client2, (const struct sockaddr*) &addr3,
+		  UV_UDP_LINUX_RECVERR);
+  ASSERT(r == 0);
+
+  if (can_recverr) {
+    r = uv_udp_recv_start(&client2, alloc_cb, recv_cb);
+    ASSERT(r == 0);
+
```

Comment:
Nit: it would be great in the new code use the new `ASSERT` macros

Reply:
Done